crates/solidity/src/process/mod.rs

```diff
@@ -74,6 +82,11 @@ pub fn call(input: Input) -> anyhow::Result<Output> {
         anyhow::anyhow!("{:?} subprocess spawning error: {:?}", executable, error)
     })?;
 
+    #[cfg(debug_assertions)]
+    if let Some(dbg_config) = &input.debug_config {
+        let _ = dbg_config.dump_stage_output(&input.contract.path, Some("stage"), &input_json);
```

Comment:
Is silently swallowing the error here intended? Intuitively should either bubble up or panic here?